lib/pbench/server/api/resources/datasets_list.py

```diff
@@ -45,45 +45,46 @@ def __init__(self, config: PbenchServerConfig, logger: logging.Logger):
             role=API_OPERATION.READ,
         )
 
-    def _get(self, json_data: JSON, request: Request) -> Response:
+    def _get(self, _, request: Request) -> Response:
         """
         Get a list of datasets matching a set of criteria.
 
         NOTE: This does not rely on a JSON payload to identify the dataset and
         desired metadata keys; instead we rely on URI query parameters.
 
+        Args:
```

Comment:
Can we move move the above `NOTE` to something like:

```
Args:
    _: The JSON data payload of the GET request, ignored
    request: The original Request object containing query parameters
             identifying the dataset and desired metadata keys
```

Reply:
The JSON payload is not part of the signature of this function (which is what the note was explaining).  Yeah, there's a parameter there (which is why the `_` is there), but this function doesn't use it.  So, we shouldn't (have) need(ed) to document it....